src/main/java/io/github/syst3ms/skriptparser/registration/SkriptRegistration.java

```diff
@@ -319,14 +330,61 @@ public void addEvent(Class<? extends SkriptEvent> c, Class<? extends TriggerCont
     }
 
     /**
-     * Starts a registration process for a {@link Type}
+     * Starts the registration process for an {@link Enum} as a {@link Type}.
+     * <br>
+     * The {@link TypeRegistrar#literalParser(Function) literalParser(Function)}
+     * and {@link TypeRegistrar#toStringFunction(Function) toStringFunction(Function)} methods
+     * are implemented at default.
+     * Note that the represented Enum's values need to be in the {@code UPPER_CASE_SNAKE} convention.
```

Comment:
Real nitpick, but I believe this is more usually referred to as SCREAMING_SNAKE_CASE